uiowa/uiowa
# [Articles] Pagination bug

Resolves https://github.com/uiowa/uiowa/issues/8049. 



# How to test
```
ddev blt ds --site=pathology.medicine.uiowa.edu && ddev drush @medicinepathology.local uli /news 
```
- Review update hooks
   - Check views block update hook - is there a better way to do this for blocks?

1. Click on page 9 in the pager
2. Confirm `Iowa Association of Pathologists Award Winners` appears at the bottom
3. Click on page 10 in the pager and confirm that `Steven Moore receives Impact Scholar award April 27, 2021 from Dean Brooks Jackson` appears at the top of the list
4. Test against https://medicinepathology.uiowa.ddev.site/articles-block with same test as above. 
5.  Test newly exposed "Sort" filter for title in the block settings
6. Are there any downsides to adding this additional sort filter?
